qemu/overlay/etc/promtail/promtail.yaml

```diff
@@ -1 +0,0 @@
-../../../../etc/promtail/promtail.yaml
```

Comment:
why is this gone ?

Reply:
This is installed from the binaries, it's not part of zos core anymore